### ISSUE TITLE
Spamming warnings about URI.unescape with ruby 2.7.0 and solargraph-0.38.6

### ISSUE THREAD

**Issue by Felixoid** [castwide/solargraph#303]
Hello. I have pretty annoying warning with the mentioned versions:

```
....
/home/felixoid/.gem/ruby/2.7.0/gems/solargraph-0.38.6/lib/solargraph/language_server/uri_helpers.rb:17: warning: URI.unescape is obsolete
/home/felixoid/.gem/ruby/2.7.0/gems/solargraph-0.38.6/lib/solargraph/language_server/uri_helpers.rb:17: warning: URI.unescape is obsolete
/home/felixoid/.gem/ruby/2.7.0/gems/solargraph-0.38.6/lib/solargraph/language_server/uri_helpers.rb:17: warning: URI.unescape is obsolete
....
```

It's referencing to [the line](https://github.com/castwide/solargraph/blob/master/lib/solargraph/language_server/uri_helpers.rb#L17) `17:        URI.decode(uri).sub(/^file\:\/\//, '').sub(/^\/([a-z]\:)/i, '\1')`

**Comment by castwide** [comment]
This issue is on the roadmap. I hope to have it fixed in the next patch release.

**Comment by castwide** [comment]
Fixed in 0.39.5. More information: #322
